lib/view_component/base.rb

```diff
@@ -495,6 +516,14 @@ def sidecar_files(extensions)
         (sidecar_files - [source_location] + sidecar_directory_files + nested_component_files).uniq
       end
 
+      def cache_on(*args)
+        class_eval <<~RUBY, __FILE__, __LINE__ + 1
+          def __vc_cache_args
```

Comment:
I'm not sure I understand why this is defining a method if we're accepting symbols as arguments.

could this be:

```ruby
def cache_on(*args)
  @__vc_cache_args ||= args
end
```

Reply:
@BlakeWilliams this has chnaged please take a look when you can 🙏